docs/changes/units/15710.bugfix.rst

```diff
@@ -0,0 +1,17 @@
+Fix rare signature incompatibilities between helper and helped array functions.
+Affected functions:
```

Comment:
Maybe good to be a bit more explicit. How about something like:
```
Most involve cases where the corresponding numpy function has had its
arguments renamed between numpy versions. Since all those generally
the first arguments, which are typically passed as positional arguments,
this should not affect code.
```